### PR TITLE
Fixes MULEbots being bloody at roundstard, Fixes #11001

### LINKED ISSUE #11001: Mulebots spawn blood-stained

### PR DESCRIPTION
**Pull request by ghost**
The examine proc was treating everything that had the potential to have blood DNA as bloodied, in this case, MULEbots
Fixes #11001

### CODE REVIEW

**Comment by WJohn** [comment]
You should put in Fixes #11001 in the OP, I'm not sure if the comments can auto-close issues like that.

File: code/game/atoms.dm [hunk 1]
```diff
@@ -231,7 +231,7 @@ its easier to just keep the beam vertical.
 /atom/proc/examine(mob/user)
 	//This reformat names to get a/an properly working on item descriptions when they are bloody
 	var/f_name = "\a [src]."
-	if(src.blood_DNA && !istype(src, /obj/effect/decal))
+	if(src.blood_DNA.len && !istype(src, /obj/effect/decal))
```
Comment (phil235) [inline code/game/atoms.dm]: I think you need to do `blood_DNA && blood_DNA.len && ...` , blood_DNA can be null afaik.

**Comment by RemieRichards** [comment]
Please address @phil235's concern, @Ahammer18

**Comment by Razharas** [comment]
@Ahammer18 plz respond

**Comment by Aranclanos** [comment]
see https://github.com/tgstation/-tg-station/pull/11526

**Pull request closed without merging**